keps/sig-api-machinery/2161-apiserver-default-labels/kep.yaml

```diff
@@ -23,9 +24,7 @@ replaces:
 stage: alpha
 latest-milestone: "v1.21"
 milestone:
-  alpha: "v1.21"
-  beta: "v1.22"
-  stable: "v1.23"
+  stable: "v1.21"
```

Comment:
shouldn't this be:
```
beta: "v1.21"
stable: "v1.22"
```

if we're allowing the feature gate to be disabled for a release?

Reply:
ok fixed